### PR TITLE
Land Use and Vegetation Suitability pyBRAT symbology 

### PR DESCRIPTION
**Pull request by Cashe93**
Tracked in #395

### CODE REVIEW

**Review by MattReimer — APPROVED** [review]
This looks good to me. 

@Cashe93 let me know if this fixes things for qrave. I think I'll wait to rebuild webRAVE tiles for pyBRAT until we have the missing file issue sorted out.

**Review by MattReimer — CHANGES_REQUESTED** [review]
Hmmm. I can't merge this because of an automated check failing

```
E       [./Symbology/web/pyBRAT/py_VEG_CODE.txt] :: Error parsing QGIS Color Ramp: list index out of range
```

This file is missing the mandatory header

**Review by jtgilbert — CHANGES_REQUESTED** [review]
In the txt files for the web there are commas that need to be removed from the names. 
In py_lu.txt 'Not Developed' is 'Not,Developed'
In py_VEG_CODE.txt 'Barely Suitable' and 'Moderately Suitable' are have commas also ('Barely,Suitable' and 'Moderately,Suitable').
Do a find and replace to get rid of the unwanted commas.

File: Symbology/web/pyBRAT/py_VEG_CODE.txt [hunk 1]
```diff
@@ -1,7 +1,3041 @@
+11,0,97,0,102,Preferred
```
Comment (MattReimer) [inline Symbology/web/pyBRAT/py_VEG_CODE.txt]: No headers. This won't work

**Review by MattReimer — CHANGES_REQUESTED** [review]
Just missing headers on one file

**Comment by Cashe93** [comment]
@MattReimer and @jtgilbert, I've just pushed commits with the requested changes.

**Review by jtgilbert — APPROVED** [review]

**Comment by Cashe93** [comment]
@MattReimer, it looks like the automated check has failed again. I've looked into it but can't figure out why. Let me know if this is something on my end and I'll get it fixed.

**Review by MattReimer — APPROVED** [review]

**Comment by MattReimer** [comment]
@Cashe93 the line looks right to me. This could be a glitch in the checking software. I'll take it from here and integrate the branch when I'm done

**Comment by MattReimer** [comment]
FOund the problem. You had a second set of header lines halfway down the file. Fixed and we can merge now.

**Pull request merged**